src/formatter.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Is programming with bare functions (ie functions that don't have associated state) in an OO style?